test/sql/function_calls.sql

```diff
@@ -820,21 +820,15 @@ begin;
         returns account language sql stable
     as $$ select id, email, name from account where id = 2; $$;
 
-    create function returns_all_columns_null_account()
-        returns account language sql stable
-    as $$ select id, email, name from account where id is null; $$;
-
     create function returns_null_account()
         returns account language sql stable
     as $$ select id, email, name from account where id = 9; $$;
 
     insert into account(id, email, name)
     values
         (1, 'aardvark@x.com', 'aardvark'),--all columns non-null
-        (2, 'bat@x.com', null),--mixed: some null, some non-null
-        (null, null, null);--all columns null
```

Comment:
after adding the pkey constraint, this record is no longer valid.

Since require a pkey I don't think it's necessary/possible to test for this case because can't happen. right?